cuda/test/base/kernel_launch.cu

```diff
@@ -51,15 +51,42 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "core/test/utils.hpp"
 
 
-namespace {
-
-
```

Comment:
the namespace is deleted

Reply:
this is an executable, not a library, so the unnamed namespace has no effect on the execution of the program

Reply:
I know. It is from gtest example and we have it everywhere in ginkgo. but it is not important.

Reply:
If there is no reason for adding it, I don't think we need to keep it. [This SO question](https://stackoverflow.com/questions/47861534/why-does-google-test-sample-put-tests-in-an-anonymous-namespace) discusses the issue, but I don't really think the answer applies to us at all. Symbol visibility is a problem that we will deal with in the future #715, but it does not have any impact here (except for maybe a miniscule difference in link time because the symbol table of the object files is bigger)